net/iperf3/files/etc/config/iperf3

```diff
@@ -0,0 +1,16 @@
+
+config server wan
+	option enabled '1'
```

Comment:
I'm not sure I fully understand the users' expectations from this package, but I'd probably not expect the server to start on wan by default.

Reply:
good point will fix with the next push and comment out this section